services/queue/src/workclaimer.js

```diff
@@ -151,21 +38,29 @@ class WorkClaimer extends events.EventEmitter {
     this._hintPollers = {}; // provisionerId/workerType -> HintPoller
   }
 
+  getHintPoller(taskQueueId) {
```

Comment:
this is extracted from `claim()` while loop. Not sure why it was done inside the loop anyway 🤷